src/tests/ch-2.spec.ts

```diff
@@ -0,0 +1,245 @@
+import { isEqual, subtract, getMonthNames, convertToUTCDate, getGenderFromSalutation } from "../utils/ch-2";
+
+describe('isEqual', () => {
+  it('returns true when two dates are equal', () => {
```

Comment:
Let's use `test` instead of `it` to describe the each single unit test
and follow the structure
`test('it should return true with parameters of equal value')`